fiatconnect-api.md

```diff
@@ -800,12 +800,7 @@ On success, the server MUST return an HTTP `200`, with the following response bo
 	},
 	kyc: {
 		kycRequired: `boolean`,
-		kycSchemas: {
-			kycSchema: `KycSchemaEnum`,
-			allowedValues: {
-				[string]: `string[]`
-			}
-		}[]
```

Comment:
I wonder if it would be worth just eliminating the "allowedValues" field for now, and migrating from a list of objects to a list of KYC schemas later on, to limit the scope of the changes that Valora and CICO providers will need to make in the near term

Reply:
I think even dropping `allowedValues` will be a breaking change since a required field will no longer be present.